core/storage/base_model/gae_models.py

```diff
@@ -510,7 +510,7 @@ class VersionedModel(BaseModel):
     # previous versions is stored in the snapshot models.
     version = ndb.IntegerProperty(default=0)
 
-    def _require_not_marked_deleted(self):
+    def require_not_marked_deleted(self):
```

Comment:
This isn't the only change that you need to make...the test file needs to be changed as well or it looses track of these methods.